source/user-manual/reference/ossec-conf/vuln-detector.rst

```diff
@@ -66,6 +70,20 @@ Enables indexing of vulnerability inventory data.
 | **Allowed values** | ``yes``, ``no``             |
 +--------------------+-----------------------------+
 
+offline-url
+^^^^^^^^^^^
+Allows to specify a path or URL for offline content access.
+
++--------------------+-----------------------------+
+| **Default**        | Empty                       |
++--------------------+-----------------------------+
+| **Allowed values** | ``<path>``, ``<URL>``       |
```

Comment:
We could use `file://<LOCAL_FILE_PATH>` and `http[s]://<URL>` instead

Reply:
I agree!